client/containers/DashboardCustomScripts/dashboardCustomScripts.scss

```diff
@@ -9,4 +9,8 @@
     .bp3-tab-panel {
         height: calc(100vh - 300px);
     }
+    .callout-text {
+        max-width: 500px;
+        line-height: 15px;
```

Comment:
let's add a little bottom padding here (20px?). It feels a little cramped.

Also, I don't think we need the max width. I did the mockup that way only because I was lazily copying and pasting UI elements in screenshots, but it looks better and more warning-y, I think, all the way across the screen.